system/Database/BaseUtils.php

```diff
@@ -260,7 +260,8 @@ public function getXMLFromResult(ResultInterface $query, array $params = []): st
      *
      * @param array|string $params
      *
-     * @return mixed
+     * @return false|string|void
+     * @phpstan-return false|string|never
```

Comment:
Can be changed to never for native return?

Reply:
Do you mean `@return false|never|string`?

Reply:
Done.